### PR TITLE
chore(build): replace esbuild with rollup

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

File: rollup.config.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        });
+    return testFolders;
+}
+
+const components = getFoldersInAFolder('./src/components');
+const input = components.reduce((inputObject, componentName) => {
+    inputObject[`components/${componentName}/${componentName}`] = path.join(process.cwd(), `src/components/${componentName}/${componentName}.ts`);
+    return inputObject;
+}, {});
+
```
Comment (yinonov) [inline rollup.config.ts:26]: should we?
```suggestion

/**
 * @type {import('rollup').RollupOptions}
 */
```

**Review by yinonov — CHANGES_REQUESTED** [review]
should rollup.config.js be .git ignored?

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+var components = getFoldersInAFolder('./src/components');
+console.log(components);
+// export default {
+//     input: 'src/index.js',
+//     output: {
+//         dir: 'output',
+//         format: 'cjs'
+//     },
+//     plugins: [typescript()]
+// };
```
[the remark this answers is not in the document]
Comment (YonatanKra) [reply rollup.config.js]: Removed :)

File: rollup.config.ts (hunk 1 above)
Comment (YonatanKra) [reply rollup.config.ts:26]: What for?

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "prestart": "tsc -b",
     "start": "tsc --watch --preserveWatchOutput & web-dev-server --watch",
     "build:typescript": "tsc -b",
-    "build": "node ./scripts/build.js",
+    "build": "rollup -c",
```
Comment (yinonov) [inline package.json:10]: I kind of like the separation of scripts to files

File: rollup.config.ts (hunk 1 above)
Comment (yinonov) [reply rollup.config.ts:26]: https://rollupjs.org/guide/en/#config-intellisense
maybe not needed in the ts case?

Comment (YonatanKra) [reply rollup.config.ts:26]: Not needed. That's why I'm using TS anywhere possible.

File: package.json (hunk 3 above)
Comment (YonatanKra) [reply package.json:10]: In the future, when we will have a good reason to do so, and the capacity to maintain scripts, I am all for it. This is why I postponed with Nx cause building the `generators` and `executors` is a big task.

File: rollup.config.ts (hunk 1 above)
Comment (YonatanKra) [reply rollup.config.ts:26]: This file is no longer in the repo BTW.

**Review by yinonov — APPROVED** [review]

**Review by yinonov — APPROVED** [review]

**Pull request merged**